### ISSUE TITLE
similarity with revive linters receiver-naming

### ISSUE THREAD

**Issue by ccoVeille** [ldez/recv#2]
https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#receiver-naming

https://github.com/mgechev/revive/blob/6a139caf92c8eaf68884fc7907cf5405b67aa0b0/rule/receiver-naming.go#L70-L78

is it OK that they could report the same errors about consistency

**Comment by ldez** [comment]
:thinking: I don't know.

As this linter is dedicated to receivers, it can be good to handle all receiver related rules.

Fun fact: I checked this rule 2 days ago to see if it was a duplicate of another linter.

**Comment by ccoVeille** [comment]
I agree about checking everything related to receivers in one linter

Then you could consider reporting `_` receivers the same way receiver-naming does. Either via an option or enabled directly.

**Comment by ccoVeille** [comment]
Also, there is this linter that is about receiver

https://github.com/mgechev/revive/blob/master/rule%2Fmodifies-value-receiver.go

It's an interesting one that helped me to spot bugs. It has only one caveat, every single code/example based on cobra. The method has a receiver and return the receiver. So it reports invalid usage on non-pointer receiver

But maybe I should simply report an issue on revive linter there.
